lib/ocpp/v201/profile.cpp

```diff
@@ -102,7 +102,7 @@ void period_entry_t::init(const DateTime& in_start, int in_duration, const Charg
     const auto start_tp = std::chrono::floor<seconds>(in_start.to_time_point());
     start = std::move(DateTime(start_tp + seconds(in_period.startPeriod)));
     end = std::move(DateTime(start_tp + seconds(in_duration)));
-    limit = in_period.limit;
+    limit = in_period.limit.value_or(0); // FIXME
```

Comment:
TODO: find out implications for backwards compatibility with 2.0.1
(limit became optional)